app-backend/Dockerfile

```diff
@@ -1,9 +0,0 @@
-FROM openjdk:8-jre
```

Comment:
This is being removed because it was being used to apply migrations pre-Flyway, right?

Reply:
I believe so: 

https://github.com/raster-foundry/raster-foundry/blob/393bb29bd0af435ab0545080bd792860b9a27cef/docker-compose.test.yml#L45-L51

I couldn't find any usage of this file in the current state of `develop`.